package.json

```diff
@@ -70,14 +71,13 @@
         "css-minimizer-webpack-plugin": "^5.0.1",
         "eslint": "^8.54.0",
         "eslint-config-prettier": "^9.0.0",
-        "eslint-loader": "^4.0.2",
```

Comment:
Denne (er deprecated for en del år siden) er hovedårsak til det meste herfra; 
https://github.com/navikt/familie-ef-ettersending/security/dependabot?q=package%3Aloader-utils+manifest%3Ayarn.lock+has%3Apatch